### ISSUE TITLE
Results not updated according to optimizations

### ISSUE THREAD

**Issue by davidfischer-ch** [dhermes/project-euler#2]
Please update the results for problem n°37, it should run 230+ times faster in python now.
https://github.com/dhermes/project-euler/blob/master/benchmark_output/side_by_side_10times.txt

**Comment by dhermes** [comment]
"Now" meaning what?

**Comment by davidfischer-ch** [comment]
Since the pull-request #1 was merged.

**Comment by dhermes** [comment]
#1 wasn't merged. Also problematic with the timings: I don't have the hardware I originally ran it on, so all times would need to be updated.

**Comment by davidfischer-ch** [comment]
Yes, but this commit c157d2f5361cf0f72ecb947279975e46d059576f was.
I had a speed-up of 232 when implementing it. So you may use this number to upgrade the results.
Its not scientific but fair enough for showing that the Python implementation is finally not that slow :)

**Comment by dhermes** [comment]
Ah now I remember. I think I'd rather wait until I have time to update all of the times.

**Comment by davidfischer-ch** [comment]
OK - Just let this issue open as a reminder :)
Thanks.

**Comment by dhermes** [comment]
SGTM
